pingpong-go/src/pingpong/pong/Pong.go

```diff
@@ -1,29 +1,35 @@
-package pong 
+package pong
 
 import (
+	"bytes"
 	"fmt"
 	"net/http"
-	"github.com/gorilla/mux"
 	"strconv"
-	"bytes"
+
+	"github.com/gorilla/mux"
 )
 
 func pongHandler(w http.ResponseWriter, r *http.Request) {
 	params := mux.Vars(r)
-	i, _ := strconv.Atoi(params["length"])
+	i, err := strconv.Atoi(params["length"])
+	if err != nil {
+		panic(err)
+	}
 	i -= 4
-	var result bytes.Buffer
-	result.WriteString("po")
-	for ; i > 0; i-- { result.WriteString("o") }
+	result := bytes.NewBufferString("po")
```

Comment:
Yes, a benchmark would be good. I am not aware, whether implementation behind the type is the same in Go?